Src/java/engine/src/test/java/org/opencds/cqf/cql/engine/runtime/DateTimeTest.java

```diff
@@ -368,4 +368,74 @@ void dateTimeEquals() {
 
         // TODO: It'd be good to extend these across different types of constructors, for example, from a string
     }
+
+    @Test
+    void roundToPrecision() {
```

Comment:
Suggestion:  try with different timezones and bonus points for a weird half-hour timezone, like America/St_Johns